recOrder/acq/acquisition_workers.py

```diff
@@ -965,109 +940,41 @@ def _save_imgs(self, birefringence, phase, meta=None):
         -------
 
         """
-        writer = WaveorderWriter(self.snap_dir)
+        prefix = self.calib_window.save_name
 
         if birefringence is not None:
 
-            # initialize
-            chunk_size = (
-                1,
-                1,
-                1,
-                birefringence.shape[-2],
-                birefringence.shape[-1],
-            )
-
-            # increment filename one more than last found saved snap
-            prefix = self.calib_window.save_name
             name = (
                 f"BirefringenceSnap.zarr"
                 if not prefix
                 else f"{prefix}_BirefringenceSnap.zarr"
             )
-
-            # create zarr root and position group
-            writer.create_zarr_root(name)
-
-            # Check if 2D
-            if len(birefringence.shape) == 3:
-                writer.init_array(
-                    0,
-                    (
-                        1,
-                        4,
-                        1,
-                        birefringence.shape[-2],
-                        birefringence.shape[-1],
-                    ),
-                    chunk_size,
-                    ["Retardance", "Orientation", "BF", "Pol"],
-                )
-                z = 0
-
-            # Check if 3D
-            else:
-                writer.init_array(
-                    0,
-                    (
-                        1,
-                        4,
-                        birefringence.shape[-3],
-                        birefringence.shape[-2],
-                        birefringence.shape[-1],
-                    ),
-                    chunk_size,
-                    ["Retardance", "Orientation", "BF", "Pol"],
-                )
-                z = slice(0, birefringence.shape[-3])
-
-            # Write the data to disk
-            writer.write(birefringence, p=0, t=0, c=slice(0, 4), z=z)
-
-            current_meta = writer.store.attrs.asdict()
-            current_meta["recOrder"] = meta
-            writer.store.attrs.put(current_meta)
+            with open_ome_zarr(
+                os.path.join(self.snap_dir, name),
+                layout="fov",
+                mode="w-",
+                channel_names=["Retardance", "Orientation", "BF", "Pol"],
```

Comment:
What is the 'Pol' channel?

Reply:
Also it's not in iohub's [channel display setting](https://github.com/czbiohub/iohub/blob/06645045d64a9b0e68614c5cbcffa8cb3abfa779/iohub/display_utils.py#L89-L100) so there is no contrast limit adjustments done here.

Reply:
`Pol` is the label for what `waveorder` calls `DoP` (degree of polarization), and what we might alternatively call "depolarization". We don't use this channel much, so it's fine that it has default display settings IMO.